### DISCUSSION TITLE
Clicking a DIV

### DISCUSSION THREAD

**Discussion by igalnassi** [ChromiumDotNet/WebView2.DevTools.Dom#24] [Q&A]
I am trying to click a DIV but it doesn’t look like the HtmlDivElement has a click method on it. What is the best way to do this?

**Comment by amaitland** [comment]
Have you tried `ClickAsync`?

```c#
var div = await DevToolsContext.CreateHtmlElementAsync<HtmlDivElement>("div");

await div.ClickAsync();
```

Otherwise you can execute any sort of javascript you like

```c#
await div.EvaluateFunctionAsync("div => div.click()");
```

**Comment by igalnassi** [comment]
I don’t think the create will work as I need to click a specific div.

I was able to figure out the latter solution though and that works.

Thank you!
